src/backend/commands/player/toxic/kill_player.cpp

```diff
@@ -27,7 +28,7 @@ namespace big
 			    10000.0f,
 			    2,
 			    0,
-			    (1 << 4),
+			    (1 << 4) | 0x80000,

```

Comment:
The compiler will optimize this out, but it should either be replaced with 0x80010 or a #DEFINE of what these bits means.